script.js

```diff
@@ -143,13 +154,75 @@ const confirmOfClearForm = () => {
     });
 };
 
+const valuesToForm = id => {
+  const values = $$("filmsDataTable").getItem(id);
+  $$("addFilmForm").setValues(values);
+};
+
+webix.DataStore.prototype.sorting.as.sortByFloat = (firstNum, secondNum) => {
+  const first = Number.parseFloat(firstNum);
+  const second = Number.parseFloat(secondNum);
+  return first > second ? 1 : first < second ? -1 : 0;
```

Comment:
Use regular 'if' instead of using ternary operator twice. It makes your code harder to read